pkg/collector/scheduler.go

```diff
@@ -134,10 +144,35 @@ func (s *CheckScheduler) getChecks(config integration.Config) ([]check.Check, er
 	checks := []check.Check{}
 	numLoaders := len(s.loaders)
 
+	initConfig := commonInitConfig{}
+	err := yaml.Unmarshal(config.InitConfig, &initConfig)
+	if err != nil {
+		return nil, err
+	}
+	selectedLoader := initConfig.LoaderName
+	log.Debugf("Get checks for check '%s' with loader name '%s'", config.Name, selectedLoader)
```

Comment:
This comment can be improved by adding details about which loader we are checking - in this case would be `initConfig`